docs/simulations.rst

```diff
@@ -42,10 +42,10 @@ from the physicsgrid file.  The names of the observed columns are
 the observed data are given).
 
 When creating simulated observations, using the standard IMF mass prior will
-skew your catalog to lower-mass stars.  If you wish to sample the full range of
-stellar masses, you can set the mass prior to `{'name': 'flat'}` and the
-age prior to `{'name': 'flat_log'}` in `datamodel.py` before creating the
-model grid.
+skew your catalog to lower-mass stars.  If you wish to sample an equal number of
```

Comment:
It is really an equal number?  I don't see how the two priors give an *equal* number.  I do see how this biases the sampling to give more high mass stars, but equal?

Reply:
Fair point.  I was paraphrasing your suggestion on the earlier iteration, but easy enough to change :)

Reply:
Thanks.